Packs/cisco-ise/Integrations/cisco-ise/cisco-ise.yml

```
@@ -309,7 +311,55 @@ script:
     - contextPath: CiscoISE.Endpoint.Link
       description: 'The link for the endpoint '
       type: String
-  dockerimage: demisto/python3:3.7.4.977
+  - arguments:
+    - default: false
+      description: 'MAC address of the endpoint (format: 11:22:33:44:55:66).'
+      isArray: false
+      name: mac_address
+      required: true
+      secret: false
+    - default: false
+      description: 'A list of custom attributes'
+      isArray: false
+      name: attributes_map
+      required: false
+    deprecated: false
+    description: Given a mac address and custom attribute list, a new Endpoint is
+      created on Cisco ISE.
+    execution: false
+    name: cisco-ise-create-endpoint
+    outputs:
+    - contextPath: CiscoISE.Endpoint.MACAddress
+      description: Mac address of the newly created Endpoint.
+      type: string
+  - deprecated: false
+    description: Returns data for all Cisco ISE nodes in the deployment.
+    execution: false
+    name: cisco-ise-get-nodes
+    outputs:
+    - contextPath: CiscoISE.NodesData
+      description: Details of all deployment ISE nodes.
+      type: string
+  - arguments:
+    - default: false
+      description: 'MAC address of the endpoint (format: 11:22:33:44:55:66).'
+      isArray: false
+      name: macAddress
```

let's keep the args in snake case
```suggestion
      name: mac_address
```

---

fixed, although the other commands do use "macAddress" in the integration